src/javascript/EditPanelContainer/EditPanel/EditPanel.utils.js

```diff
@@ -164,10 +175,10 @@ export function getDynamicFieldSetOfField(sections, fieldName) {
  * @returns boolean value, true if the node has mixin, false otherwise.
  */
 function hasNodeMixin(node, mixin) {
-    return node.mixinTypes && node.mixinTypes.find(mixinType => mixinType.name === mixin);
+    return node && node.mixinTypes && node.mixinTypes.find(mixinType => mixinType.name === mixin);
 }
 
-function getMixinsToMutate(nodeData = {}, formValues = {}, sections) {
```

Comment:
I don't know why you remove this, I guess it protect us from null value.

Reply:
It has been remove to have better control of our data